formasaurus/classifiers.py

```diff
@@ -93,14 +125,32 @@ def load(cls, filename=None, autocreate=True, rebuild=False):
 
         """
         if filename is None:
-            filename = cls._cached_model_path()
-
-        if rebuild or (autocreate and not os.path.exists(filename)):
+            if env_path := os.environ.get("FORMASAURUS_MODEL"):
+                filename = os.path.expanduser(env_path)
+            else:
+                file_name = _default_model_file_name()
+                filename = at_root("data", file_name)
```

Comment:
```suggestion
                filename = at_root("data", "model")
```
It seems unnecessary to have _default_model_filename function. Also, it's better to avoid having both `file_name` and `filename` variables :)